lib/vm-migration/migrate.js

```diff
@@ -414,6 +414,28 @@ function migrateVm(req, res, next) {
             }
 
             if (action === 'begin' || action === 'estimate') {
+                // Validate override params.
+                if (req.params.override_server_uuid &&
```

Comment:
What about merging both into a single check and avoid duplicated code/wording?:
```
['override_server_uuid', 'override_uuid'].forEach(function ensureValidUuid(p) {
    if (req.params[p] && !common.validUUID(req.params[p]){
        cb(new errors.ValidationFailedError(util.format(
            'Invalid parameters: %s (%s ) is not a valid UUID', p, req.params[p]),
            [errors.invalidUuidErrorsElem(p)] ));
        return;
    }
});
```

Reply:
I tried this - but I think it ends up messier, due to the fact that the cb() is now in an inner function - so then you'd need to add a break in the inner function, add a marker that the uuid was invalid and then have a check/return in the outer function, i.e.:
```
                var hasInvalidUuid = false;
                ['override_server_uuid', 'override_uuid'].forEach(
                        function ensureValidUuid(p) {
                    if (req.params[p] && !common.validUUID(req.params[p])) {
                        cb(new errors.ValidationFailedError(util.format(
                            'Invalid parameters: %s (%s) is not a valid UUID',
                            p, req.params[p]),
                            [ errors.invalidUuidErrorsElem(
                                'override_server_uuid') ]
                        ));
                        hasInvalidUuid = true;
                        break;
                    }
                });
                if (hasInvalidUuid) {
                    return;
                }
```

This approach could also work using a for loop (no inner scope), so I'll try that and also move it into the `validateMigrationBegin` call - where it probably should have been in the first place.